index.html

```diff
@@ -151,6 +151,25 @@ <h1>My text</h1>
                 'option': 1, 'script': 1, 'style': 1, 'textarea': 1, 'title': 1
               };
 
+              const htmlTags = {
```

Comment:
I think we can use the previously defined objects here, like:

```javascript
const htmlTags = {
  ...voidTags,
  ...textTags,
  a: 1, abbr: 1, // and so on
}
```

Reply:
Nice catch; I just changed it